### PR TITLE
[Feat] -  Check for command

### PR DESCRIPTION
**Pull request by jenaye**

### CODE REVIEW

**Comment by jenaye** [comment]
maybe should use function like this 


```go
func commandExists(cmd string) bool {
	_, err := exec.LookPath(cmd)
	color.Red("Please define command : %v  in ure path => %q", cmd, err)
	color.Red("you can use alias like this : dirsearch='python /home/<user>/softs/dirsearch/dirsearch.py'")
	return false
}
```

**Review by darkweak — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -115,6 +120,20 @@ func main() {
 	version := figure.NewColorFigure("Yelaa v.1.1", "", "cyan", true)
 	version.Print()
 
+	_, err := exec.LookPath("dirsearch")
+	if err != nil {
```
Comment (darkweak) [inline main.go:124]:
```suggestion
	if commandExists("dirsearch") {
```

File: main.go [hunk 2]
```diff
@@ -115,6 +120,20 @@ func main() {
...
+		color.Red("Please define Dirsearch in ure path => %q", err)
+		color.Red("you can use alias like this : dirsearch='python /home/<user>/softs/dirsearch/dirsearch.py'")
+		os.Exit(0)
+	}
+
+	_, err = exec.LookPath("nuclei")
+	if err != nil {
```
Comment (darkweak) [inline main.go:131]: Same

**Review by darkweak — CHANGES_REQUESTED** [review]

File: main.go [hunk 3]
```diff
@@ -115,6 +120,20 @@ func main() {
 	version := figure.NewColorFigure("Yelaa v.1.1", "", "cyan", true)
 	version.Print()
 
+	_, err := exec.LookPath("dirsearch")
+	if err != nil {
+		color.Red("Please define Dirsearch in ure path => %q", err)
+		color.Red("you can use alias like this : dirsearch='python /home/<user>/softs/dirsearch/dirsearch.py'")
+		os.Exit(0)
```
Comment (darkweak) [inline main.go:127]: It should exit with the status 1

**Review by darkweak — CHANGES_REQUESTED** [review]

**Pull request merged**